packages/flutter_webrtc/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: flutter_webrtc_tizen
+description: Flutter WebRTC plugin for Tizen, based on flutter_webrtc project.
+version: 0.0.1
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+  flutter: '>=1.22.0'
```

Comment:
Is flutter version valid? Other plugins are 'flutter: ">=2.0.0"'.

Reply:
Hmm.. It's part from the original code. What are you concerned about?

Reply:
https://github.com/flutter-tizen/plugins/runs/4637769778?check_suite_focus=true#step:9:1018
I am not sure, but is this error related to the version?

Reply:
Hmm... that's strange.. I don't know if it's a problem like that, but I'll check it out

Reply:
Oh~ I see. Thank you.